src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
 // write code here
+
+function toNumber(Str) {
```

Comment:
```suggestion
function convertToNumber(str) {
```
Consider naming functions according to these rules
![image](https://user-images.githubusercontent.com/96409563/230307869-1b36c4b6-ba37-4983-8e0f-6d043fcae12f.png)
There is also no need to capitalize the parameter